hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ClientRemoveMigrationListenerCodec.java

```diff
@@ -34,9 +34,9 @@
  */
 
 /**
- * Removes a Migration listener to the cluster.
+ * Removes the specifiec migration listener.
```

Comment:
Looks like there is a typo on the protocol side. `specifiec`  -> `specified` 
I know it is not related to your work but would you mind fixing that for us?

Reply:
Fixed.

Reply:
Reverted back because it leads to test failures.

Reply:
Fixed through protocol generator.